uaa/src/test/java/org/cloudfoundry/identity/uaa/scim/endpoints/ScimUserEndpointsMockMvcTests.java

```diff
@@ -1067,6 +1067,68 @@ public void testCreateUserWithEmailDomainNotAllowedForOriginUaa() throws Excepti
         }
     }
 
+    @Test
+    public void testCreateUserTxSuccess() throws Exception {
+        ScimUser user0 = createScimUser("user0");
+        ScimUser user1 = createScimUser("user1");
+        ScimUser user2 = createScimUser("user2");
+        ScimUser user3 = createScimUser("user3");
+        ScimUser[] users = {user0, user1, user2, user3};
+        MockHttpServletRequestBuilder postUsersTx = post("/Users/tx")
+                .header("Authorization", "Bearer " + scimCreateToken)
+                .contentType(APPLICATION_JSON)
+                .content(JsonUtils.writeValueAsBytes(users));
+
+        ResultActions resultActions = getMockMvc().perform(postUsersTx)
+                .andExpect(status().isCreated());
+
+        ScimUser[] createdUsers = JsonUtils.readValue(resultActions.andReturn().getResponse().getContentAsString(), ScimUser[].class);
+
+        for(ScimUser user : createdUsers ){
+            getAndReturnUser(HttpStatus.OK.value(), user, scimReadWriteToken);
+        }
+    }
+
```

Comment:
Can you add a test for if you are doing a batch create but one of the users already exists in the database? It would be very similar to the duplicate keys test except the duplication would not occur in the same payload.